.travis.yml

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 rvm: 2.3.3
+=======
+rvm: 2.5.3
+>>>>>>> f149fe0... Change ruby version to 2.5.3
 
```

Comment:
I don’t think you intended to have diff markers in this commit?

Reply:
No, sorry. I jumped down a rabbit hole. Fixing now.